### ISSUE TITLE
Correct build commend in Comet Development Guide

### ISSUE THREAD

**Issue by whynick1** [apache/datafusion-comet#305]
### Describe the bug

```
make test-java: compile the project and run tests in Java side
```
in [DEVELOPMENT.md](https://github.com/apache/datafusion-comet/blob/main/DEVELOPMENT.md#build--test) seems should be
```
make test-jvm: compile the project and run tests in Java side
```
according to https://github.com/apache/datafusion-comet/blob/main/Makefile#L31.

**Comment by viirya** [comment]
Thank you @whynick1

**Comment by planga82** [comment]
fixed in #364
